WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -715,6 +719,29 @@ class MySiteViewModel @Inject constructor(
         }
     }
 
+    fun performFirstStepAfterSiteCreation(siteLocalId: Int) {
+        if (landOnTheEditorABExperiment.getVariation() is Control) {
+            checkAndStartQuickStart(siteLocalId)
+        } else {
+            checkAndStartLandOnTheEditor()
+        }
+    }
+
+    fun checkAndStartLandOnTheEditor() {
+        if (landOnTheEditorABExperiment.getVariation() is Control) {
+            return
+        }
```

Comment:
Curious why we are checking this twice?

Reply:
Good catch @mkevins 👍 
The call hierarchy was initially different and this made sense but now it is not needed.
Fixed with 96e08c48ac637088e9072f14fd4ffc43adb25615